### PR TITLE
added inactive tags on group cards

### PR DESCRIPTION
**Pull request by oratiu**

### CODE REVIEW

**Review by m-doughty — APPROVED** [review]
Looks good.

File: packages/react-components/src/organisms/__tests__/GroupCard.test.tsx [hunk 1]
```diff
@@ -21,6 +22,13 @@ it('renders the group name linking to the group', () => {
   );
 });
 
+it('renders the state tag for a inactive group', () => {
```
Comment (kmaid) [inline packages/react-components/src/organisms/__tests__/GroupCard.test.tsx:25]: please test for the absense of inactive in this test with a rerender

File: apps/storybook/src/GroupCard.stories.tsx [hunk 2]
```diff
@@ -12,5 +12,17 @@ export const Normal = () => (
...
+);
+
+export const Inactive = () => (
+  <GroupCard
+    id="42"
+    name={text('Name', 'My Group')}
+    description={text('Description', 'Group Description')}
+    tags={array('Tags', ['Tag 1', 'Tag 2'])}
+    numberOfTeams={number('Number of Teams', 3)}
+    active={false}
```
Comment (kmaid) [inline apps/storybook/src/GroupCard.stories.tsx:26]: Could this just be a boolean knob?

Comment (oratiu) [reply apps/storybook/src/GroupCard.stories.tsx:26]: It could 👍

File: packages/react-components/src/organisms/__tests__/GroupCard.test.tsx (hunk 1 above)
Comment (oratiu) [reply packages/react-components/src/organisms/__tests__/GroupCard.test.tsx:25]: On it

File: packages/react-components/src/organisms/GroupCard.tsx [hunk 3]
```diff
@@ -2,38 +2,56 @@ import { css } from '@emotion/react';
...
+    alignItems: 'flex-start',
+  },
+});
+
 type GroupCardProps = Pick<
   GroupResponse,
   'id' | 'name' | 'description' | 'tags'
 > & {
   readonly numberOfTeams: number;
+  readonly active?: boolean;
```
Comment (kmaid) [inline packages/react-components/src/organisms/GroupCard.tsx:34]: Its this not a property you can pick off group response?

**Review by kmaid — APPROVED** [review]

File: packages/react-components/src/organisms/GroupCard.tsx [hunk 4]
```diff
@@ -2,38 +2,56 @@ import { css } from '@emotion/react';
...
   paddingRight: `${15 / perRem}em`,
 });
 
+const titleStyle = css({
+  display: 'flex',
+  flexFlow: 'row',
+  gap: `${16 / perRem}em`,
+  alignItems: 'center',
+  marginBottom: `${-12 / perRem}em`,
+  [`@media (max-width: ${tabletScreen.width - 1}px)`]: {
```
Comment (kmaid) [inline packages/react-components/src/organisms/GroupCard.tsx:22]: You should try and develop mobile first. This is like putting in the mobile view changes afterwards. gap: 0 i think is invalid I think you wanted unset.

**Pull request merged**